types/node/module.d.ts

```diff
@@ -1,6 +1,43 @@
 declare module "module" {
     import { URL } from "url";
-    namespace Module {}
+    namespace Module {
+        /**
+         * Updates all the live bindings for builtin ES Modules to match the properties of the CommonJS exports.
+         * It does not add or remove exported names from the ES Modules.
+         */
+        function syncBuiltinESMExports(): void;
+
+        /**
+         * @experimental
+         */
+        function findSourceMap(path: string, error?: Error): SourceMap;
```

Comment:
Shouldn’t&nbsp;these be&nbsp;defined as&nbsp;static&nbsp;members of&nbsp;the&nbsp;`Module`&nbsp;class?

Reply:
They should be both but similarly to `events` the exports are hard to model without duplicating a bunch of things.
It's also not _really_ intended to be used as a static but rather just slapped on the module for exports.
See https://github.com/nodejs/node/blob/master/lib/module.js#L7
For that reason I have decided to not model it as such.